package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Rubik's Cube Timer for the Terminal-buff speedsolver",
   "main": "index.js",
   "scripts": {
-    "test": "exit 0"
+    "test": "mocha"
```

Comment:
We can't use Mocha right now because I didn't develop this module with testing in mind. I am not sure tests can be incorporated without major changes in the architecture (there are several small functions but they are not testable). In any case, I am not sure I want tests. So, for this PR, please remove this change.

If you are interested in adding tests, we can discuss the merits in an issue.